### ISSUE TITLE
scan_time and time_increment are 0

### ISSUE THREAD

**Issue by DavideLanzoniSacmi** [PepperlFuchs/pf_lidar_ros2_driver#4]
**Describe the bug**
Looking at the content of the message published on the topic /scan, the values scan_time and time_increment are 0

**To Reproduce**
1. Run `ros2 launch pf_driver r2000.launch.py`
2. Run `ros2 topic echo /scan`
3. See values:
`time_increment: 0.0
scan_time: 0.0
`

**Expected behavior**
These two values should represent the time between two points readings, and the total time of 360° reading.

**Environment (please complete the following information):**
 - OS: Ubuntu 22.04 LTS
 - ROS Version: ROS2 humble

**Sensor**
 - Device:  R2000 UHD
 - FW Version: 1.61
 - HW Version: 1.73

**Comment by kawk** [comment]
Yes, for R2000, it shouldn't be zero but `scan_time = 1/scan_frequency` and `time_increment = scan_time/samples_per_scan`. Additionally, for R2300, the `time_increment` is 1/90kHz (independent of scan frequency) but I'm unsure about the correct value for `scan_time` for 4-layer R2300, as it publishes `LaserScan` data on four separate topics, one per layer, so their `scan_time` probably should be `4/scan_frequency`?
